db/warehouse/migrate/20230505150822_add_custom_data_element_definitions.rb

```diff
@@ -0,0 +1,26 @@
+class AddCustomDataElementDefinitions < ActiveRecord::Migration[6.1]
+  def up
+    create_table :CustomDataElementDefinitions do |t|
+      # Describe what this custom field should be associated with
+      t.string :owner_type, null: false, index: true, comment: 'Record that this type of data element must be associated with'
+      t.references :custom_service_type, index: true, comment: 'Service type that this type of data element must be associated with'
```

Comment:
Not sure if we will want/need this, but we'll see. We can remove it if it doesn't end up being used, but I think it will be helpful if we can say: "there is a custom field called Amount $$ that is collected for service types x,y,z" especially since definitions are linked to service types